### ISSUE TITLE
Translate GOVERNANCE.md

### ISSUE THREAD

**Issue by margaritis** [nodejs/iojs-el#9]
It might be difficult for Greek folks to follow the Governance doc so i believe it would make sense to translate it. 

P.S. Your opinion on that @sirodoht, @AntouanK @fanixk?

**Comment by sirodoht** [comment]
Agreed. I could see it in the next 2-3 days if there no one else available to translate.
